charts/runner-manager/values-runner-manager-gcloud.yaml

```diff
@@ -0,0 +1,57 @@
+gcloudServiceAccount: "vault:secret/data/runner-manager#GCLOUD_SERVICEACCOUNT"
+githubToken: "vault:secret/data/runner-manager#GITHUB_TOKEN"
+redhatUsername: "vault:secret/data/runner-manager#REDHAT_USERNAME"
+redhatPassword: "vault:secret/data/runner-manager#REDHAT_PASSWORD"
+#redis:
+#  password: "vault:secret/data/runner-manager#rREDIS_PASSWORD"
+#allowedSshKeys: "vault:secret/data/runner-manager#allowed_ssh_keys"
+githubOrganization: scalanga-devl
+deployment:
+  ingress:
+    #enabled: true
+    #tls: true
+    #fqdn: "gcloud.runner-manager.scality.net"
+    #tlsSecret: 'gcloud-runner-manager-cert'
+    #annotations:
+    #  cert-manager.io/cluster-issuer: "letsencrypt-prod"
+    #  kubernetes.io/ingress.class: nginx
+    # basicAuthCreds: "vault:secret/data/runner-manager#WEBHOOK_HTPASSWD"
+serviceAccount: eve-ci@scality-devl.iam.gserviceaccount.com
+cloudName: gcloud
+cloudConfig:
+  project_id: "scality-prod-ga-runners"
+  zone: us-east1-c
+timeoutRunnerTimer:
+  minutes: 20
+  hours: 0
+pythonConfigModule: 'srcs.settings.settings_local'
+serviceMonitor:
+  enabled: true
+  labels:
+    monitoring-instance: prometheus-central
+
+image:
+  tag: '0.4.0'
```

Comment:
You should make another release if you push new files

Reply:
Ok